README.md

```diff
@@ -146,7 +146,7 @@ suite of acceptance tests against it. The tests are written in ruby, using
 capybara and rspec. To run the tests, you first need to install a few
 dependencies:
 
-    $ sudo apt install gcc g++ make qt5-default libqt5webkit5-dev ruby-dev zlib1g-dev
+    $ sudo apt install gcc g++ make qt5-default libqt5webkit5-dev ruby-dev xvfb zlib1g-dev
```

Comment:
As we discussed in the issue, mind breaking `xvfb` off and saying below this line something like "If you don't have X running, you'll need a virtual framebuffer as well: (provide the install command)."